src/main.css

```diff
@@ -371,6 +388,116 @@ button[disabled] {
   background: none;
   border: none;
 }
+.ui-datepicker-close {
+	display: none;
+}
+.ui-widget-content button {
+	float: right !important;
+}
+
+.ui-datepicker thead {
+    border-bottom: unset;
+}
+
+.ui-datepicker th {
+    background: white;
+    font-weight: 500;
+}
+
+.ui-datepicker table {
+    width: 100%;
+    font-size: .9rem;
+    border-collapse: collapse;
+    border: 2px solid #f2f2f2;
+    border-top: 0;
+}
+
+#ui-datepicker-div .ui-state-default, #ui-datepicker-div .ui-widget-content .ui-state-default{
+    margin: .1em;
+}
+
+.ui-datepicker td a, .ui-datepicker td span {
+    display: block;
+    padding: .2em;
+    text-align: center;
+    text-decoration: none;
+}
+
+.ui-datepicker .ui-datepicker-prev-hover, .ui-datepicker .ui-datepicker-next-hover {
+    top: 2px;
+}
+
+.ui-datepicker .ui-datepicker-prev-hover {
+    left: 2px;
+}
+
+.ui-datepicker .ui-datepicker-next-hover {
+    right: 2px;
+}
+
+/* override timepicker styles */
+.ui-timepicker-hours {
+	padding: 0;
+	background: white;
+}
+.ui-timepicker-minutes {
+	padding: 0;
+	background: white;
+}
+.ui-corner-all {
+	border-radius: unset;
+}
+.ui-timepicker-title {
+	text-align: center;
+}
+.ui-timepicker td {
+	border: none;
+}
+
+.ui-timepicker th {
+    background: none;
+    border-bottom:unset;
+}
+.ui-timepicker td a {
+	display: block;
+	padding: 3px 5px;
+	width: 19px;
+	text-align: right;
+	text-decoration: none;
+	cursor: pointer;
+}
+.ui-timepicker-hours .ui-timepicker tbody {
+    border-right: 2px solid #f2f2f2;
+    border-left: 2px solid #f2f2f2;
+    border-bottom: 2px solid #f2f2f2;
+}
+
+.ui-timepicker-minutes .ui-timepicker tbody {
+    border-right: 2px solid #f2f2f2;
+    border-bottom: 2px solid #f2f2f2;
+}
+
+#ui-timepicker-div, #ui-datepicker-div {
+    font-size: 16px;
+}
+
+#ui-datepicker-div .ui-icon-circle-triangle-e, #ui-timepicker-div .ui-icon-circle-triangle-e {
+    background-image: url(data:image/svg+xml;charset=utf8;base64,PHN2ZyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnIHZpZXdCb3g9Jy0yMjk2NSAtMjExMTguOTQxIDE0LjgyIDI0Jz48cGF0aCBmaWxsPSd3aGl0ZScgZD0nTTguNTksMjcuMTgsMTcuNzUsMTgsOC41OSw4LjgyLDExLjQxLDZsMTIsMTItMTIsMTJaJyB0cmFuc2Zvcm09J3RyYW5zbGF0ZSgtMjI5NzMuNTkgLTIxMTI0Ljk0MSknLz48L3N2Zz4=) !important;
+    background-position: initial;
+}
+
+#ui-datepicker-div .ui-icon-circle-triangle-w, #ui-timepicker-div .ui-icon-circle-triangle-w {
+    background-image: url(data:image/svg+xml;charset=utf8;base64,PHN2ZyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnIHZpZXdCb3g9Jy0yMzM5OC4yOTUgLTIxMTE4Ljk0MSAxNC44MiAyNCc+PHBhdGggZmlsbD0nd2hpdGUnIGQ9J004LjU5LDI3LjE4LDE3Ljc1LDE4LDguNTksOC44MiwxMS40MSw2bDEyLDEyLTEyLDEyWicgdHJhbnNmb3JtPSd0cmFuc2xhdGUoLTIzMzc0Ljg4NSAtMjEwODguOTQxKSByb3RhdGUoMTgwKScvPjwvc3ZnPg==) !important;
+    background-position: initial;
+}
+
+.ui-icon-circle-triangle-w:after {
+    content: unset;
+}
+
+.ui-icon-circle-triangle-e:after {
+    content: unset;
+}
```

Comment:
The issue with overriding the jQuery styles in main is that it's a lot of effort to maintain (every time binary-style changes we need to update this). 

We should look for a way to migrate away from jQuery UI? (but that's out of the scope for this card 😄 )